crates/polars-ops/src/chunked_array/strings/substring.rs

```diff
@@ -115,3 +158,35 @@ pub(super) fn substring(
         _ => ternary_elementwise(ca, offset, length, substring_ternary),
     }
 }
+
+pub(super) fn head(ca: &StringChunked, n: &Int64Chunked) -> StringChunked {
+    match (ca.len(), n.len()) {
+        (_, 1) => {
+            // SAFETY: index `0` is in bound.
```

Comment:
Should be "in bounds". Or my preference:
```suggestion
            // SAFETY: We have verified the series has at least 1 element.
```

Same for the other safety comments.

Reply:
I went with the more specific:

```rust
// SAFETY: `ca` was verified to have at least 1 element.
```

replacing `ca` with the relevant variable in each case. This seems a bit safer to me in case code is moved around, as it ties the comment to the unsafe code. I updated `str.slice` above that as well.